tooling/linux_repro_build_compare.sh

```diff
@@ -0,0 +1,143 @@
+#!/bin/sh
+# shellcheck disable=SC2006,SC3010,SC2046,SC2166,SC2086
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+# This script examines the given SBOM metadata file, and then builds the exact same binary
+# and then compares with the Temurin JDK for the same build version, or the optionally supplied TARBALL_URL.
+
+set -e
+
+[ $# -lt 1 ] && echo Usage: $0 SBOMURL TARBALL_URL && exit 1
+SBOMURL=$1
+TARBALL_URL=$2
+ANT_VERSION=1.10.5
+ANT_CONTRIB_VERSION=1.0b3
+
+installPrereqs() {
+  if test -r /etc/redhat-release; then
+    yum install -y gcc gcc-c++ make autoconf unzip zip alsa-lib-devel cups-devel   libXtst-devel libXt-devel libXrender-devel libXrandr-devel libXi-devel
+    yum install -y file fontconfig fontconfig-devel systemtap-sdt-devel # Not included above ...
+    yum install -y git bzip2 xz openssl pigz which # pigz/which not strictly needed but help in final compression
+    if grep elease.6 /etc/redhat-release; then
+      if [ ! -r /usr/local/bin/autoconf ]; then
+        curl https://ftp.gnu.org/gnu/autoconf/autoconf-2.69.tar.gz | tar xpfz - || exit 1
+        (cd autoconf-2.69 && ./configure --prefix=/usr/local && make install)
+      fi
+    fi
+  fi
+}
+
+# ant required for --create-sbom
+downloadAnt() {
+  if [ ! -r /usr/local/apache-ant-${ANT_VERSION}/bin/ant ]; then
+    echo Downloading ant for SBOM creation:
+    curl https://archive.apache.org/dist/ant/binaries/apache-ant-${ANT_VERSION}-bin.zip > /tmp/apache-ant-${ANT_VERSION}-bin.zip
+    (cd /usr/local && unzip -qn /tmp/apache-ant-${ANT_VERSION}-bin.zip)
+    rm /tmp/apache-ant-${ANT_VERSION}-bin.zip
+    echo Downloading ant-contrib-${ANT_CONTRIB_VERSION}:
+    curl -L https://sourceforge.net/projects/ant-contrib/files/ant-contrib/${ANT_CONTRIB_VERSION}/ant-contrib-${ANT_CONTRIB_VERSION}-bin.zip > /tmp/ant-contrib-${ANT_CONTRIB_VERSION}-bin.zip
+    (unzip -qnj /tmp/ant-contrib-${ANT_CONTRIB_VERSION}-bin.zip ant-contrib/ant-contrib-${ANT_CONTRIB_VERSION}.jar -d /usr/local/apache-ant-${ANT_VERSION}/lib)
```

Comment:
Need to check a checksum here?

Reply:
these are just helper script for users, so not here. If they want to ensure the version then they will have already installed it at /usr/local/apache-ant-${ANT_VERSION}/bin/ant